implementations/rust/ockam/ockam_identity/src/secure_channel/handshake/handshake_worker.rs

```diff
@@ -40,6 +41,7 @@ pub(crate) struct HandshakeWorker {
     role: Role,
     remote_route: Option<Route>,
     decryptor_handler: Option<DecryptorHandler>,
+    flag: Arc<AtomicBool>,
```

Comment:
Every variable name is an opportunity you have to give hints to the reader about your intention, try to give a more meaningful name that reflects what you had in mind, even if imperfect.